### DISCUSSION TITLE
The URL information is lost, resulting in a large number of 404 incoming httpx

### DISCUSSION THREAD

**Discussion by hktalent** [projectdiscovery/naabu#344] [General]
The URL information is lost, resulting in a large number of 404 incoming httpx

https://xxx1.com/sitectx/ Status code is 200

echo https://xxx1.com/sitectx/ | naabu -silent | httpx -silent

But when it comes to httpx, because sitectx cannot be obtained, the detection can only get 404

**Comment by ehsandeep** [comment]
@hktalent naabu is port scanner, **dns** and **ip/cidr** is expected and valid input for it, `https://xxx1.com/sitectx/ ` is url and invalid input to feed for port scanner.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
